teabot_endpoints/models.py

```diff
@@ -1,9 +1,20 @@
+import os
+
 from peewee import Model, DateTimeField, OperationalError, CharField, \
-    IntegerField, ForeignKeyField, BooleanField
+    IntegerField, ForeignKeyField, BooleanField, PostgresqlDatabase
 from playhouse.sqlite_ext import SqliteExtDatabase
 from datetime import datetime
 
-db = SqliteExtDatabase('teapot.db')
+DATABASE_URL = os.environ.get("DATABASE_URL", "")
+if DATABASE_URL.startswith("postgres"):
```

Comment:
We'll need to reimport that data from our sqlite file to Postgres, is this easy enough?